sonic_api.yaml

```diff
@@ -2474,6 +2474,10 @@ definitions:
         type: integer
         format: int32
         description: vxlan id to be used for the tunnel. Optional arg. If it isn't provided the default vxlan id defined for the vnet/vrf will be used as the destination vnid.
+      persistent:
```

Comment:
I was wondering what would be the expected behavior if there are requests for the same route prefix in both persistent and non-persistent mode? Also if there are entries for the same route prefix in both config_db and appl_db with the same distance, which one should take effect?

Reply:
I did clarify that in the description. In normal cases, its not expected, but if it happens, persistent takes precedence